Assets/Scripts/UI/UIScrollers/HorizontalScroller.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+using UnityEditor;
+using System.IO;
+
+public class HorizontalScroller : MonoBehaviour
```

Comment:
Deze class heet `HorizontalScroller` maar het blijkt ook dat hij de verantwoordelijkheid heeft over de levels laden en laten zien. Split deze class op in ongeveer 3 classes, dan houden we ons aan de `single responsibility principle`.